test/membrane/rtp/vad_test.exs

```diff
@@ -0,0 +1,113 @@
+defmodule Membrane.RTP.VADTest do
+  use ExUnit.Case
+
+  alias Membrane.RTP.VAD
+
+  ExUnit.Case.register_attribute(__MODULE__, :clock_rate)
```

Comment:
Do we need all these ExUnit attributes functionality if in fact we are not using it? Or am I missing something? To be honest I see this for the first time and seems super useful but I guess not in this case?

Reply:
For one thing, I wanted to introduce this since I was pretty sure it was new to others.

I would like to write other tests that actually override the defaults and verify the outcomes... unfortunately I still don't quite understand the code enough to write those tests.

Reply:
Ok, I just managed to write a few more tests that both exercise these test attributes, and cover a few more aspects of the module.